drivers/dma/dma_stm32.c

```diff
@@ -257,14 +257,16 @@ static int dma_stm32_configure(struct device *dev, u32_t id,
 		return -EINVAL;
```

Comment:
Commit message is unclear compared to changes.
Something like "restrict mem2mem limitation to V1"... would be clear (if I understood the change correctly).

Reply:
changed: no restriction on peripheral to/from memory dma transfer